src/presenters/admin/indexation-warning-presenter.php

```diff
@@ -125,14 +132,33 @@ private function get_incomplete_indexation_alert() {
 		$output .= \esc_html__( 'It looks like an indexing process was run earlier, but didn\'t complete. There is still some content which hasn\'t been indexed yet. Don\'t worry, you can pick up where you left off.', 'wordpress-seo' );
 		$output .= '</p>';
 		$output .= $this->get_estimate();
-		$output .= \sprintf(
+		$output .= $this->get_action( \esc_html__( 'Continue processing and speed up your site now', 'wordpress-seo' ) );
+
+		return $output;
+	}
+
+	/**
+	 * Generates the action, which is either a button or a link.
+	 *
+	 * @param string $text The text of the action.
+	 *
+	 * @return string The action.
+	 */
+	protected function get_action( $text ) {
+		if ( $this->action_is_link ) {
+			return \sprintf(
+				'<a class="button" href="%1$s">%2$s</a>',
+				\admin_url( '/admin.php?page=wpseo_tools#start-indexation-yoastIndexationData' ),
+				$text
```

Comment:
I would do the `esc_html` here, just to be sure. Now you don't know if the output is escaped.